packages/tatum-ada/src/ledger/account.ts

```diff
@@ -30,4 +31,4 @@ export {
   getAccountsByCustomerId,
   getAllAccounts,
   getAccountBalance,
-} from '@tatumio/tatum-core'
+} from '@tatumio/tatum-ledger'
```

Comment:
Should I remove these re-exports or is that necessary to have them here in specific chain module?